src/buildHTML.js

```diff
@@ -696,22 +697,26 @@ groupTypes.accent = function(group, options) {
         // shift the accent over to a place we don't want.
         accent.italic = 0;
 
+        accentBody = makeSpan(["accent-body"], [accent]);
+
+        // CSS defines `.katex .accent .accent-body { width: 0 }`
+        // so that the accent doesn't contribute to the bounding box.
+        // We need to shift the character by its width (effectively half
+        // its width) to compensate.
+        let left = -accent.width / 2;
+
+        // Shift the accent over by the skew.
+        left += skew;
+
         // The \H character that the fonts use is a combining character, and
-        // thus shows up much too far to the left. To account for this, we add a
-        // specific class which shifts the accent over to where we want it.
+        // thus shows up much too far to the left. To account for this, we add
+        // a manual shift of the width of one space.
         // TODO(emily): Fix this in a better way, like by changing the font
-        let accentClass = null;
         if (group.value.label === '\\H') {
-            accentClass = "accent-hungarian";
+            left += 0.5;  // width of space
```

Comment:
I don't think this will work for all fonts, but since we're going to be adding non-combining double acute accents to all fonts soon it's okay to leave this as is for now.

Reply:
Also, I don't think this comment is correct as the width of a space is 250 (or thereabouts depending on the font).

Reply:
Ah, right, maybe it's double the width of a space. (I was copy/pasting the comment from the `accent-hungarian` CSS.) It might also be the width of the character, which is 0.5...